cmake/YarpIDL.cmake

```diff
@@ -193,7 +193,7 @@ endfunction()
 function(YARP_IDL_TO_DIR)
 
   # Flag to control whether IDL generation is allowed.
-  option(ALLOW_IDL_GENERATION "Allow YARP to (re)build IDL files as needed" OFF)
+  option(ALLOW_IDL_GENERATION "Allow YARP to (re)build IDL files as needed" ON)
```

Comment:
This value should not be modified

Reply:
Fixed in [3c5625e](https://github.com/robotology/yarp/pull/3149/commits/3c5625e2c8a430da81a5a96e593e305145f7ae02)